docs/settings/config/channels.mdx

```diff
@@ -0,0 +1,204 @@
+---
+id: channels
+title: Channel Configuration
+sidebar_label: Channels
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Channels config options are: Index, Roles, and Settings. Channel config uses an admin message sending a `Channel` protobuf which also consists of a `ChannelSettings` protobuf.
+
+:::info
+**Channel Settings** (as described on this page) should not be confused with [LoRa Radio Settings](/docs/settings/config/lora)
+
+[LoRa Radio Settings](/docs/settings/config/lora) contain the modem configuration (frequency settings, spreading factor, bandwidth, etc.) used for the radio. These settings are identical for all channels and can *not* be unique per channel. Radio settings are directly applied to the Primary channel from which all Secondary channels derive their radio settings.
+
+**Channel Settings** contain information for segregating message groups, configuring optional encryption, and enabling or disabling messaging over internet gateways. These settings *are* unique and configurable per channel.
+:::
+
+## Channel Config Values
+
+### Index
+
+The channel index begins at 0 and ends at 7.
+
+_Indexing_ can not be modified.
+
+|        Index         |     Channel       | Default Role | Purpose | 
+| :--------------------: | :----------------: | :----------------: | :----------------: |
+| 0 | 1 | `PRIMARY` | Used as `default` channel |
+| 1 | 2 | `DISABLED` | User defined |
+| 2 | 3 | `DISABLED` | User defined |
+| 3 | 4 | `DISABLED` | User defined |
+| 4 | 5 | `DISABLED` | User defined |
+| 5 | 6 | `DISABLED` | User defined |
+| 6 | 7 | `DISABLED` | User defined |
+| 7 | 8 | `SECONDARY` | Used as `admin` channel |
```

Comment:
minor detail here, you can't have disabled channels in between and the last slot is not reserved. admin channel is any channel named "admin" (all lowercase).